src/plugins/cosmos/components/modals/Staking/Staking.tsx

```diff
@@ -39,9 +39,25 @@ import { UnstakeBroadcast } from './views/UnstakeBroadcast'
 import { UnstakeConfirm } from './views/UnstakeConfirm'
 
 const StakingModalContent = ({ assetId, validatorAddress }: StakingModalProps) => {
+  const [validateAllSteps, setValidateAllSteps] = useState(false)
   const location = useLocation()
   const translate = useTranslate()
 
+  const isBroadcast = matchPath(location.pathname, {
```

Comment:
Having some variable specifically for broadcast sounds a bit flaky, I'd prefer a way to systematically have the "step passed with green check" logic.